File: XML/Voluntario/PythonDOM/Library.py
class Library:
    def __init__(self, id, address):
        self.id = id
        self.address = address
        self.books = []
    
    def getId(self):
        return self.id
    def setId(self, id):
        self.id = id
    
    def getAddress(self):
        return self.address
    
    def setAddress(self, address):
        self.address = address
    
    def setBooks(self, book):
        self.books.append(book)           
    
    def getBook(self, index):
        return self.books[index]   
    def sizeBooks(self):
        return len(self.books)
    
    def __str__(self):
        return f"Id: {self.id}, Dirección: {self.address}, Libros: {self.showBooks()}"
    
    def showBooks(self):
        return [str(book) for book in self.books]

File: XML/PracticaSAX/src/Library.java
import java.util.LinkedList;
import java.util.List;

public class Library {
    
    private int id;
    private String address;
    private List<Book> books;

    
    public Library(){
        books = new LinkedList<>();
    }
    
    public Library(int id, String address) {
        this.id = id;
        this.address = address;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public int sizebooks(){
        return books.size();
    }
    public Book getbook(int i){
        return books.get(i);
    }
    public void addbook(Book b){
        books.add(b);
    }

    @Override
    public String toString() {
        return "Library [id=" + id + ", address=" + address + ", books=" + books + "]";
    }

    
    
}
